genesis/src/genesis_accounts.rs

```diff
@@ -227,10 +230,14 @@ fn add_stakes(
         .sum::<u64>()
 }
 
+/// Add MainnetBeta accounts that should be present in genesis; do nothing for other cluster types
 pub fn add_genesis_accounts(genesis_config: &mut GenesisConfig, mut issued_lamports: u64) {
+    if genesis_config.cluster_type != ClusterType::MainnetBeta {
```

Comment:
I don't remember the exact history of launching each of the different clusters, but I know these genesis accounts do exist on Devnet. Should we also include (escape) Devnet, for historical-record purposes?
(I assume that Testnet was launched with a different genesis because of the original adversarial test cluster use-case.)

```suggestion
    if genesis_config.cluster_type != ClusterType::MainnetBeta && genesis_config.cluster_type != ClusterType::Devnet {
```

Reply:
Alternately, we could go with this, which I think is the main thing we're trying to accomplish, right?
```suggestion
    if genesis_config.cluster_type == ClusterType::Development {
```

Reply:
> I don't remember the exact history of launching each of the different clusters, but I know these genesis accounts do exist on Devnet

Huh, never realized this but spot checks of `solana-ledger-tool genesis --acounts` would agree with you, although there are some slight differences.
```
HBGF5ySKGzsdmKkVBvLtyEBMJKLrUKgJT4sgvyEdq4pf
  balance: 1 827 645.990 454 56 SOL <== mnb
  balance: 1 827 645.518 054 56 SOL <== devnet
```
Don't feel like opening that can of worms at the moment, but to your point, I think bailing out for `Development` seems valid. As-is, I don't think you can run `solana-genesis` with only adjusting CLI flags to recreate mnb/testnet/devnet genesis.bin files

Reply:
Cool, let's just bail for `Development`, then.

Reply:
Yep, pushed as a separate commit / update PR title / rebased to tip of master for good measure